roswell/run-parachute.ros

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh
+#|-*- mode:lisp -*-|#
+#|
+exec ros -Q -- $0 "$@"
+|#
+
+;cmucl crashes with silent on
+(ql:quickload '(:ci-utils :ci-utils/coveralls :parachute :iterate)
```

Comment:
Is all this stuff necessary? Seems to just be pointlessly polluting the test environment.

Reply:
ci-utils/coveralls wraps cl-coveralls when the user requests coverage data be sent to Coveralls, and avoids loading all of cl-coveralls dependencies if not needed (there are 8 first order dependencies, with some second and third order dependencies).
ci-utils itself isn't directly needed (although loaded by ci-utils anyways), so I can remove that quickload.
For Iterate, I can refactor away the dependency, it just makes the code simpler when taking a variable number of arguments each iteration.

Reply:
I'm a bit confused why coveralls is needed at all. Shouldn't coverage be determined when someone works on tests, rather than every time CI runs for whatever reason?

Reply:
Any time when branches in the execution paths are added or refactored, it's good to double check that there isn't a case that isn't tested.  I've always see coveralls used with reports from every CI run in order to watch for degradation in code coverage more closely.  So, I wanted to make sure that style of CI is supported.

This script only measures coverage when the `COVERALLS` environmental variable is set, so the overhead is limited for workflows that don't want continuous coverage checks.

Reply:
I see, that's fair enough.